src/inverseRobot.js

```diff
@@ -8,6 +8,27 @@
 
 function inverseRobot(robot) {
   // write code here
+  let tempKyesRobot = {};
+  let tempValueRobot = {};
+
+  tempKyesRobot = Object.keys(robot);
+  tempValueRobot = Object.values(robot);
+
+  for (let i = 0; i < tempValueRobot.length; i++) {
+    if (tempValueRobot.indexOf(tempValueRobot[i]) !== i) {
+      return null;
+    }
+  }
+
+  for (const key in robot) {
+    delete robot[key];
+  }
+
+  for (const i in tempKyesRobot) {
+    robot[tempValueRobot[i]] = tempKyesRobot[i];
```

Comment:
The variable 'i' in the for...in loop is misleading because it suggests an index, but it's actually a key. Use a more descriptive variable name, such as 'keyIndex' or 'key'.